tests/ProxyTest.php

```diff
@@ -254,6 +254,15 @@ public function test_indexphp_requests_are_proxied_correctly()
         $this->assertEquals($expected, $responseBody);
     }
```

Comment:
Could you modify the `test_indexphp_requests_are_proxied_correctly` test so it also tests the URL rewriting code you added?